fabric/src/test/java/TestRegex.java

```diff
@@ -357,6 +358,27 @@ public void IngredientAnnotator_INGREDIENT_PATTERN() {
         p.shouldMatch("§7Contorted Stone§3 [§b✫✫✫§3]");
     }
 
+    @Test
+    public void LocationUtils_STRICT_COORDINATE_PATTERN() {
+        PatternTester p = new PatternTester(LocationUtils.class, "STRICT_COORDINATE_PATTERN");
+        System.out.println(p.pattern.pattern());
+        p.shouldMatch("10, 14, -1945");
+        p.shouldMatch("10591, 106, 20489");
+        p.shouldMatch("[200, 0, 0]");
+        p.shouldMatch("102, 100, 102");
+        p.shouldMatch("1002, 100, 1002");
+        p.shouldMatch("10002, 100, 10002");
+        p.shouldMatch("0, 0, 0");
+        p.shouldMatch("0, 0");
+        p.shouldNotMatch("100");
+        p.shouldNotMatch("0");
+        p.shouldNotMatch("200,000");
+        p.shouldNotMatch("195,000,000");
+        p.shouldNotMatch("20,000");
+        p.shouldNotMatch("2,000");
+        p.shouldNotMatch("");
```

Comment:
Maybe also add something like:
```
        p.shouldNotMatch("10591,106,2489");
```
if we agree that having the whitespace mandatory is a good thing.